sklearn/tree/tests/test_tree.py

```diff
@@ -1258,6 +1258,18 @@ def test_only_constant_features():
         assert est.tree_.max_depth == 0
 
 
+def test_almost_constant_feature():
```

Comment:
Could you add a reference to the PR here, maybe something like "Non regression test for ..."?

Reply:
Thanks for the review!

> Could you add a reference to the PR here, maybe something like "Non regression test for ..."?

Will do.

> And/or is there a "direct" test we could have to check that the constant was initialised correctly?

I agree a more direct way to test it would be more suitable. But it seems I cannot access this constant from python layer directly so we would have to add a cpdef method or something similar to expose it to the python layer. If you think its necessary, I will add such method but it might be an overkill for this.